### DISCUSSION TITLE
Is it possible to create a new release?

### DISCUSSION THREAD

**Discussion by ericlifs** [margelo/react-native-blurhash#152] [Q&A]
Hey! 

Can we create a new release? We are having some Android build issues which are fixed by these two commits but they are not released yet: 

<img width="595" alt="image" src="https://user-images.githubusercontent.com/7376892/157068630-e6e691af-c7e0-4364-afd0-72c7364bf625.png">

Thank you in advance,

Eric.

**Comment by mrousavy** [comment]
Hi! Yes, will do asap. Got covid now so I'm not on pc
